substrate/frame/system/src/offchain.rs

```diff
@@ -489,8 +489,14 @@ pub trait CreateSignedTransaction<LocalCall>:
 }
 
 /// Interface for creating an inherent.
+///
+/// It can also be used to create an unsigned transaction as they are both the same extrinsic
+/// variant: `Bare`. Unsigned transaction are deprecated in favor of general transaction.
```

Comment:
What do you mean by that 2nd phrase?

Reply:
The extrinsic format is has 3 variants: signed, general and bare.
signed and general both have call + additional data for transaction extensions +signature (if signed, not if general), bare only has a call.

there is different kind of extrinsic: inherent, unsigned transaction: they use the bare format. signed and general transaction they use their respective format.
We want to deprecate unsigned transaction, people can already use general transaction instead, but the PR with `pallet::authorize` will help the transition.
Looking back we shouldn't have pushed the deprecation so fast.

there is some doc here: https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/reference_docs/extrinsic_encoding/index.html